test/e2e/sriovoperatornodepolicy_test.go

```diff
@@ -32,7 +32,7 @@ var _ = Describe("Operator", func() {
 		Eventually(func() *cluster.EnabledNodes {
 			sriovInfos, _ = cluster.DiscoverSriov(clients, testNamespace)
 			return sriovInfos
-		}, timeout, interval).ShouldNot(BeNil())
+		}, Timeout*15, RetryInterval*20).ShouldNot(BeNil())
```

Comment:
With this change, sriovnetworknodestate will be reconciled by config daemon only after the controller set the spec (empty or other)

reason for timeout increase is for controller to resync nodestate with the actual spec and allow config daemon to reconcile.

see
https://github.com/k8snetworkplumbingwg/sriov-network-operator/issues/305

Reply:
@rollandf question : do we need this now that we are updating sync status immediately to success on generation 1 ?